src/containers/MainPlotContainer/index.tsx

```diff
@@ -202,18 +207,23 @@ class MainPlotContainer extends React.Component<MainPlotContainerProps> {
 
         return (
             <React.Fragment>
-                <Popover
-                    placement="right"
-                    content={popover}
-                    visible={!!popover}
-                    {...{
-                        // props not in ant.d component, but do exist
-                        // needed to style this component since it's out of the DOM structure
-                        id: "thumbnail-popover",
-                    }}
+                <MouseFollower
+                    ref={this.popoverContainer}
+                    pageX={mousePosition.pageX}
+                    pageY={mousePosition.pageY}
                 >
-                    <MouseFollower pageX={mousePosition.pageX} pageY={mousePosition.pageY} />
-                </Popover>
+                    <Popover
+                        placement="right"
+                        content={popover}
+                        open={!!popover}
+                        getPopupContainer={() => this.popoverContainer.current || document.body}
+                        {...{
+                            // props not in ant.d component, but do exist
+                            // needed to style this component since it's out of the DOM structure
+                            id: "thumbnail-popover",
+                        }}
+                    ></Popover>
```

Comment:
teeny tiny optional thing:
```suggestion
                    />
```